dts/bindings/ipc/zephyr,ipc-icmsg.yaml

```diff
@@ -21,6 +21,18 @@ properties:
     required: true
     type: phandle
 
+  dcache-alignment:
+    type: int
+    description: |
+      Data cache alignment. If any side of the communication uses cache on
+      rx-region/tx-region this property must be the biggest value of the data
+      cache line size for both sides of the communication. If no side of the
```

Comment:
Does "line" mean an invalidation unit, a write-back unit, or the greater of these two? In some systems these units are not equal

Reply:
Wasn't aware of this. It's the greatest of those two. I will update the description.

Reply:
Fixed.